.gitignore

```diff
@@ -5,6 +5,8 @@
 
 #ignore backups
 *~
+.idea/vcs.xml
```

Comment:
I know that these files should be checked in from intellij but mine is a bit different and forces me to commit it before I checkout a different branch. If someone *really* wants to check in their vcs.xml he should simply force add it.

Also .vscode always adds the folder everywhere (also subfolders). I added it in this regards

Reply:
In what way is it different? If the IDE adds one or more lines like this:
```
    <mapping directory="$PROJECT_DIR$/build/googletest-src" vcs="Git" />
```
simply remove them and maybe mark the directory `$PROJECT_DIR$/build` (`build` may be something different) as excluded. This happens, because our CMake checks out the googletest source (recommended this way). The IDE thinks, we might want to commit to this repo, but we don't so we can just remove the line.

Reply:
Thank you for this workaround.
I have no problem with doing that but every developer who joins elektra will always need to do that which is kind of unnecessary in my opinion, especially if the vcs.xml does not change at all anyway in the repository. What do you think @markus2330

Reply:
I just saw you didn't remove the file from the repository, only added it to .gitignore. That is definitely a bad solution. We should remove it completely, if we don't want the file.

Reply:
Of course we want to make the entrance of new developers as smooth as possible. @kodebach is right about .gitignore.

Reply:
By removing this file from the project the same problem will remain. Whoever uses CLion has to manually exclude the .idea folder to avoid git always trying to add the file. The file being checked in is ok as [jetbrains](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) suggests. But any changes to it should not be checked in anymore